generated/nidaqmx/constants.py

```diff
@@ -925,12 +949,12 @@ class UnitsPreScaled(Enum):
     INCHES_PER_SECOND_SQUARED = 12471  #: Inches per second per second.
     METERS_PER_SECOND = 15959  #: Meters per second.
     INCHES_PER_SECOND = 15960  #: Inches per second.
-    PA = 10081  #: Pascals.
+    PASCALS = 10081  #: Pascals.
```

Comment:
I like this

Reply:
`SoundPressureUnits.PA` is used in nidaqm-python repo. If replaced this `UnitsPreScaled.PA` will affect `SoundPressureUnits`. Thus have retained the enum value name (`PA`).